### PR TITLE
Update DOW schema to reflect table changes

### PR DESCRIPTION
**Pull request by zachclark-ccpo**
## Overview

There was a need for more data nesting (see [PR 70 ](https://github.com/dod-ccpo/atat-snow/pull/70)on atat-snow),
to support these changes the DOW schema needed to be updated.

- Removed RequiredServices schema
- Added SelectedServiceOffering
- Added ClassificationInstances

### Example schema
```
{
  "award_history": [
    {
      "contract_award_type": "INITIAL_AWARD",
      "modification_order": 0,
      "effective_date": "2022-05-26"
    }
  ],
  "contract_information": {
    "contract_number": "string",
    "current_contract_exists": true,
    "contract_expiration_date": "2022-05-26",
    "incumbent_contractor_name": "string",
    "task_order_number": "string",
    "previous_task_order_number": "string"
  },
  "to_title": "string",
  "scope": "string",
  "scope_surge": 0,
  "current_environment": {
    "current_environment_exists": true,
    "environment_instances": [
      {
        "instance_name": "string",
        "classification_level": {
          "classification": "U",
          "impact_level": "IL2"
        },
        "instance_location": "CSP",
        "csp_region": "CSP_A",
        "performance_tier": "string",
        "pricing_model": "ON_DEMAND",
        "pricing_model_expiration": "2022-07-01",
        "operating_system_licensing": "string",
        "number_of_vCPUs": 0,
        "storage_type": "string",
        "storage_amount": 0,
        "storage_unit": "GB",
        "memory_amount": 0,
        "memory_unit": "GB",
        "data_egress_monthly_amount": 0,
        "data_egress_monthly_unit": "GB"
      }
    ],
    "additional_info": "string"
  },
  "selected_service_offerings": [
    {
      "service_offering": {
        "service_offering_group": "ADVISORY",
        "name": "string"
      },
      "classification_instances": [
        {
          "usage_description": "string",
          "selected_periods": [
            {
              "period_type": "BASE",
              "period_unit_count": 0,
              "period_unit": "DAY",
              "option_order": 0
            }
          ],
          "classification_level": {
            "classification": "U",
            "impact_level": "IL2"
          },
          "need_for_entire_to_duration": true
        }
      ],
      "other_service_offering": "string"
    }
  ],
  "period_of_performance": {
    "base_period": {
      "period_type": "BASE",
      "period_unit_count": 0,
      "period_unit": "DAY",
      "option_order": 0
    },
    "option_periods": [
      {
        "period_type": "BASE",
        "period_unit_count": 0,
        "period_unit": "DAY",
        "option_order": 0
      }
    ],
    "pop_start_request": true,
    "requested_pop_start_date": "2021-07-01",
    "time_frame": "NO_LATER_THAN",
    "recurring_requirement": true
  },
  "gfe_overview": {
    "dpas_unit_id": "string",
    "property_custodian_name": "string",
    "dpas_custodian_number": "string",
    "property_accountable": true,
    "gfe_or_gfp_furnished": true
  },
  "contract_considerations": {
    "packaging_shipping_none_apply": true,
    "packaging_shipping_other": true,
    "packaging_shipping_other_explanation": "string",
    "potential_conflict_of_interest": true,
    "conflict_of_interest_explanation": "string",
    "contractor_provided_transfer": true,
    "contractor_required_training": true,
    "required_training_services": {
      "name": "string",
      "value": "string"
    }
  },
  "section_508_accessibility_standards": {
    "pii_present": true,
    "work_to_be_performed": "string",
    "system_of_record_name": "string",
    "FOIA_city_apo_fpo": "string",
    "FOIA_country": "string",
    "FOIA_street_address_1": "string",
    "FOIA_street_address_2": "string",
    "FOIA_address_type": "FOREGIN",
    "FOIA_state_province_code": "string",
    "FOIA_full_name": "string",
    "FOIA_email": "user@example.com",
    "FOIA_zip_postal_code": "string",
    "BAA_required": true,
    "potential_to_be_harmful": true,
    "section_508_sufficient": true,
    "accessibility_reqs_508": "string"
  }
}
```

### CODE REVIEW

**Review by juliusfitzhugh-ccpo — APPROVED** [review]
Looks good according to the changes from https://github.com/dod-ccpo/atat-snow/pull/70.

File: hoth_api.yaml [hunk 1]
```diff
@@ -476,28 +476,32 @@ components:
...
             $ref: "#/components/schemas/Period"
-        service_offerings:
-          type: array
-          items:
-            $ref: "#/components/schemas/ServiceOffering"
-        other_service_offering:
-          type: string
+        classification_level:
+          $ref: "#/components/schemas/ClassificationLevel"
+        need_for_entire_to_duration:
```
Comment (jasonburkert-ccpo) [inline hoth_api.yaml:503]: I was asked to make this small change on schema side for clarity
Might be good to make here as well

```suggestion
        need_for_entire_task_order_duration:
```

**Review by jasonburkert-ccpo — APPROVED** [review]
Updates look good.  I left one minor suggestion.

**Comment by zachclark-ccpo** [comment]
image
@jasonburkert-ccpo Im going to keep `need_for_entire_task_order_duration` boolean because it aligns to this part of the DOW.

**Pull request merged**